### ISSUE TITLE
Write a 5-year "chronicle" retrospective about MoFo 

### ISSUE THREAD

**Issue by OpenMatt** [MozillaFoundation/plan#566]
## Goal
* **Look back on the first 5 years of MoFo.** Focused on: why learning and leadership are the lifeblood of our community and what makes it great.
* **In the style of:** “a cover feature story in Wired/Atlantic/NY Times Magazine,” written by a journalist. 
* **What it will look like:** a colorful, concise, broad but loosely structured 5-year retrospective follow-up to http://learningfreedomandtheweb.org/
* **What this will help us do:** append to the Mozilla learning plan as background, retrospective, context-setting and accomplishment

### Roles
* Decision-maker: @msurman 
* Project manager: @openmatt
* Responsible: @benrito 
* Contractor: not yet hired
* Consulted / Informed: @KevZawacki @HPaulJohnson @benrito @bhuvanshri @thornet + brett gaylor

### Project brief & notes
https://foundation.etherpad.mozilla.org/chronicle

## Next steps
- [ ] Hire a contractor to work on it. 

### Timelines
Could be a series of deliverables / iterations, phased over time: 
- The MozFest blogs (contractor, now)
- Simple business plan inputs on impact (Bhuvan?, now)
- Press story we write (contractor, January?)

**Comment by OpenMatt** [comment]
@benrito What's the first deadline for this work? Do we need a deliverable on this for Oct 15 board meeting [phase 2]? Or not until the December board meeting [phase 3]?

**Comment by OpenMatt** [comment]
@benrito @KevZawacki are you guys taking the lead on sourcing and hiring a contractor for this? Who's job is that?

**Comment by OpenMatt** [comment]
## First deadline for this work: Oct 6 
For the Oct 15 board meeting, we need to be able to talk about:
* Our last 5 years of work
* What we learned / what impact we have had
* How that informs what we're proposing to do going forward

This is really the basic narrative. Mark and Ben and could white board some of this in Toronto. It's not that hard, but it needs to be good. And then post-board meeting (or sooner) we need to pressure test and back fill that narrative w/ details.

**Comment by KevZawacki** [comment]
@OpenMatt Ben and I have identified a likely writer — Christine Prefontaine — and will hopefully be chatting with her next week.

**Comment by thornet** [comment]
I suggest we anchor the stories as people profiles, highlighting Mark and his legacy so far at MoFo, as well as other leaders in our networks and programs. Inspiration: the WIRED feature of Joi Ito and the Media Lab.

http://blog.crismanphoto.com/inventing-the-future-part-i-the-cover/

**Comment by cassiemc** [comment]
cc @cassiemc

**Comment by OpenMatt** [comment]
@KevZawacki great! so are we blocked on progress until she is hired and on board? Anything else we can / should do before then?

**Comment by OpenMatt** [comment]
+1 @thornet

**Comment by benrito** [comment]
We’re not blocked on progress; we still have to flesh out a project plan and an interview schedule.

I’ve gotten started on a tracker document to support this work: https://docs.google.com/spreadsheets/d/1Kai28T3rdKqT6fNxqY7InBQy1WJA701lKjxYs50wpRM/edit#gid=0

What’s this group’s preference for getting more granular on the work. Sub-tickets? Conversation in this thread?

**Comment by KevZawacki** [comment]
This thread works for me. If it starts to get unwieldy, we can splinter into smaller tickets.

**Comment by cassiemc** [comment]
@OpenMatt The design brief might be helpful to glance at at this point. If you're thinking forward to design work, we'll need that to get started. http://bit.ly/mozdesignbrief

**Comment by thornet** [comment]
@benrito using this tix for tracking also works for me

**Comment by OpenMatt** [comment]
from @chrislarry33 :This is very rich. Pomax on thimble (pride. patience, org knowledge galore)
http://pomax.github.io/1442700129236/mozilla-released-a-tool-i-am-excited-about

**Comment by OpenMatt** [comment]
@benrito what's the next step here? status? anything I can do to help?

**Comment by benrito** [comment]
We’ve been populating all the tabs in

https://docs.google.com/spreadsheets/d/1Kai28T3rdKqT6fNxqY7InBQy1WJA701lKjxYs50wpRM/edit#gid=1919120141

Feel free to pitch in?

Looking to schedule a Ben/Kevin meeting sometime Friday to finalize the roadmap in tab #2. Let us know if you want to join.

Bad news: our preferred writer may not work out, but we’re looking for alternates.

**Comment by OpenMatt** [comment]
paging Dr. @brettgaylor -- I think he has ideas?

**Comment by brettgaylor** [comment]
One person I might recommend is Julia Lipscomb. She interviewed me for The Awl and is a freelancer, based in NYC.
http://www.theawl.com/2015/04/track-and-yield

**Comment by OpenMatt** [comment]
@benrito helpful? 
* Moving to Oct 9

**Comment by OpenMatt** [comment]
We're blocked on hiring a journalist. @benrito are you going to do that?

**Comment by KevZawacki** [comment]
@OpenMatt @benrito We're meeting with our first writer on Thursday on next week, and have an email out to a tentative second writer.

**Comment by OpenMatt** [comment]
@KevZawacki thanks Kevin. So just so we're clear: are we saying that the completed article is *not* going to be done in time for the Oct 14 board meeting?

**Comment by KevZawacki** [comment]
@OpenMatt Correct. We will likely have a handful of individual profile completed by Oct. 14, but the full project (>20 interviews + photos + design) won't wrap until early 2016.

**Comment by benrito** [comment]
We will have a separate bit of analysis for the October board meeting. It
will be based on pattern read out of the "grant funded projects" tab.

**Comment by OpenMatt** [comment]
@Benrito: what about putting some of this Chronicle stuff through the filter of the 'strong non-profits' framework.

E.g. Where were we when I started or in 2010 re:

1. Serve and advocate
2. Make markets work
3. Inspire evangelists
4. Nurture (nonprofit) networks
5. Master the art of adaptation
6. Share leadership

... and where are we now?

**Comment by KevZawacki** [comment]
Update on the writer front: I spoke with Christine and Julia today, and both are on board and excited to get started. I'll be speaking with two additional writers — Maia and Anya — tomorrow.

@benrito & @xmatthewx: Next step is getting a contract to Christine and Julia so they can start interviewing and writing. Any progress on that front? The contract can draw from this [doc](https://docs.google.com/document/d/1YWkj0cHcDNgPZ_p8mT2vkSFCWfSB4zlDHwRi40T_AmU/edit).

**Comment by chrislarry33** [comment]
awsm news!! We are in finishing the process of collecting data on all our leadership development programs. When we are done we will make sure this team has as research for the writers. Good stuff in there. @thornet @leahatplay @ldecoursy will signal when ready.

**Comment by KevZawacki** [comment]
@cassiemc Quick design brief question! Should Ben and I go ahead and fill out a first draft, or is that something we do on a call with the designer(s)?

**Comment by cassiemc** [comment]
@KevZawacki We can do it either way, whatever makes sense for you. If there's a lot of context you'd like to pull together in the brief beforehand that is always helpful.

**Comment by benrito** [comment]
Think that Mark already did this in his latest slides. Let’s confirm.

**Comment by OpenMatt** [comment]
Moving this to next heartbeat

**Comment by benrito** [comment]
Which part? The graphic satisfies the Board deadline; we would need subtickets if we want to track progress toward the January release.

**Comment by ldecoursy** [comment]
@OpenMatt Can we close this?

**Comment by hannahkane** [comment]
What about now? Can we close?

**Comment by benrito** [comment]
If you want—we will file more granular tickets for next HB

**Comment by hannahkane** [comment]
could also move this ticket to current heartbeat. I'm just trying to close out old heartbeats. it's up to you.

**Comment by OpenMatt** [comment]
sounds like @benrito is on the hook for this one. Will close and leave with him.

**Comment by jlin** [comment]
Going to put the referenced etherpad at https://public.etherpad-mozilla.org/p/foundationchronicle

**Comment by benrito** [comment]
We decided to put this on hold for now but the profiles we’ve done so far will make their way into blogs, newsletters and other places.

We may want to revive this for MozFest later in the year.
